app-conf/FetcherConf.xml

```diff
@@ -32,6 +32,8 @@
   <fetcher>
     <applicationtype>mapreduce</applicationtype>
     <classname>com.linkedin.drelephant.mapreduce.MapReduceFetcherHadoop2</classname>
+    <!-- If encounter performance issues, try to use fetcher below -->
```

Comment:
I think this should be rephrased. It could be something like: "The MapReduceFSFetcherHadoop2 improves performance by eliminating the job history server as a middle man and pulling job data directly from HDFS. If you're having performance issues, it may help.".